deployments/local-minio/configs/mis_config.yml

```diff
@@ -8,4 +8,4 @@ solr:
   host: solr
   port: 8983
 endpointUrl:
-  retrieve: http://multi-int-store:8080/mis/product/
\ No newline at end of file
+  retrieve: http://localhost:9041/mis/product/
```

Comment:
Ah! OK. I misunderstood how we were using that property.

Reply:
Actually, you're right. We should send http://multi-int-store:8080/mis/product/ to the transform endpoint, but the query endpoint should return uris with `http://localhost:9041/mis/product/`. I think we should keep this change until we integrate with a transform service for local deployment because it makes testing easier. At that point, we can address the issue of the local hostnames.